draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1283,33 +1283,39 @@ multiple invocations of hash\_to\_base, by factoring out the common computation.
 
 ## Implementation {#hashtobase-impl}
 
-The following procedure implements hash\_to\_base.
+The following procedure implements hash\_to\_base. 
 
 ~~~
 hash_to_base(msg, ctr)
 
 Parameters:
-- H, a cryptographic hash function producing b bits.
+- DSS, a domain separation string chosen according to the
+  guidelines given in {{domain-separation}}.
+- H, a cryptographic hash function.
 - F, a finite field of characteristic p and order q = p^m.
-- W = ceil((ceil(log2(p)) + k) / b), where k is the security
-  parameter of the cryptosystem (e.g., k = 128).
+- L = ceil((ceil(log2(p)) + k) / 8), where k is the security parameter
+  of the cryptosystem (e.g., k = 128).
```

Comment:
Super nitpicky (and totally my fault): maybe break "parameter" onto the next line to keep line lengths approximately equal?

:sweat:

Reply:
Not your fault! Good catch.